tutor/commands/jobs.py

```diff
@@ -173,6 +173,9 @@ def importdemocourse(
 # Import into CMS
 python ./manage.py cms import ../data "$course_root"
 
+# Work around bug where outline is not generated
+python ./manage.py cms update_course_outline course-v1:OpenedX+DemoX+DemoCourse
```

Comment:
There's another issue with the proposed fix, which is that it does not work when importing courses other than course-v1:OpenedX+DemoX+DemoCourse.

AFAIU this is not a bug in Tutor, but in edx-platform, and it should be fixed there. We can cherry-pick the change here if necessary.